site/docs/data.md

```diff
@@ -5,20 +5,12 @@ title: Data
 permalink: /docs/data.html
 ---
 
-Akin to [Vega](https://www.github.com/vega/vega)'s [data model](https://www.github.com/vega/vega/wiki/Data), the basic data model used by Vega-Lite is *tabular* data, similar to a spreadsheet or a database table. Individual data sets are assumed to contain a collection of records, which may contain any number of named data fields.
-
-{: .suppress-error}
-```json
-{
-  "data": ... ,       // data
-  "mark": ... ,
-  "encoding": ... ,
-  ...
-}
-```
+Akin to [Vega](https://www.github.com/vega/vega)'s [data model](https://vega.github.io/vega/docs/data/), the basic data model used by Vega-Lite is *tabular* data, similar to a spreadsheet or a database table. Individual data sets are assumed to contain a collection of records, which may contain any number of named data fields.
 
 Vega-Lite's `data` property describes the visualization's data source as part of the specification, which can be either [inline data](#inline) (`values`) or [a URL from which to load the data](#url) (`url`).  Alternatively, we can create an empty, [named data source](#named) (`name`), which can be [bound at runtime](https://vega.github.io/vega/docs/api/view/#data).
 
+Upon ingest, Vega assigns each data object a unique id property, accessible via a custom [Symbol](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol). As a result, the id property is not accessible via a string key and is not enumerable, though you can observe the id value when inspecting data objects in a JavaScript console.
```

Comment:
This is something about all the docs so it should be in one place. I want to keep the docs lean so that people actualy read it.